src/common/components/Inspiration/Inspiration.module.css

```diff
@@ -0,0 +1,7 @@
+.inspirationArea{
+
+}
+.inspirationContainer{
+    border: 1px solid var(--blueBorder);
```

Comment:
This is an example of how to use a global color. 
We should be using global colors Always